src/output-area/model.ts

```diff
@@ -72,19 +62,19 @@ class OutputBaseModel {
 export
 class DisplayDataModel extends OutputBaseModel {
   /**
-  * The raw data for the output.
-  */
+   * The raw data for the output.
+   */
   data: MimeBundle;
 
   /**
-  * Metadata about the output.
-  */
+   * Metadata about the output.
+   */
   metadata: any;
 
   /**
-   * Output type
+   * The output type.
    */
-  outputType: OutputType = OutputType.DisplayData;
+  outputType: "display_data";
```

Comment:
This is only a declaration. It's not an initialization. You have to use `=` and actually assign it something or the actual value will be `undefined`.